query_optimizer/ExecutionGenerator.cpp

```diff
@@ -993,13 +995,15 @@ void ExecutionGenerator::convertInsertTuple(
 
   insert_destination_proto->set_insert_destination_type(S::InsertDestinationType::BLOCK_POOL);
   insert_destination_proto->set_relation_id(input_relation.getID());
+  insert_destination_proto->mutable_layout()->CopyFrom(
+      input_relation.getDefaultStorageBlockLayout().getDescription());
 
   const vector<block_id> blocks(input_relation.getBlocksSnapshot());
   for (const block_id block : blocks) {
-    insert_destination_proto->AddExtension(S::BlockPoolInsertDestination::blocks, block);
+    insert_destination_proto->AddExtension(
+        S::BlockPoolInsertDestination::blocks, block);
```

Comment:
This change is note related to this PR, and for the code, we have `120` char limits.